cypress/e2e/moderation.cy.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  addComment,
+  closeModerationPlugin,
+  openModerationPlugin, openPopoverMenu,
+  reportAndVerifyRequest,
+  openAndSelectItemFromDropdown
+} from "./utils";
+
+const MANIFEST = `#EXTM3U
+#EXT-X-MEDIA:TYPE=AUDIO,GROUP-ID="audio",LANGUAGE="en",NAME="English",AUTOSELECT=YES,DEFAULT=YES,URI="${location.origin}/media/index_1.m3u8",SUBTITLES="subs"
+
+#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=509496,RESOLUTION=480x272,AUDIO="audio",SUBTITLES="subs"
+${location.origin}/media/index.m3u8`;
+
+const moderateItems = [
+    {id: 1, label: 'Sexual Content'},
+    {id: 2, label: 'Violent Or Repulsive'},
+    {id: 3, label: 'Harmful Or Dangerous Act'},
+    {id: 4, label: 'Spam / Commercials'}
+];
+
+describe('Moderation plugin', () => {
+  beforeEach(() => {
```

Comment:
check if we can use before

Reply:
I tested moving the content of beforeEach outside of the describe and using before instead, only the first test passed. looks like the preparation is required before each test.